app/viewModels/transport/border/active/AddAnotherBorderTransportViewModel.scala

```diff
@@ -57,8 +59,15 @@ object AddAnotherBorderTransportViewModel {
             }
 
             val changeRoute = controllers.transport.border.active.routes.IdentificationController.onPageLoad(departureId, mode, index).url
-            val removeRoute = Some(controllers.transport.border.active.routes.RemoveBorderTransportYesNoController.onPageLoad(departureId, mode, index).url)
-
+            val removeRoute =
+              if (
+                index.isFirst
+                && (userAnswers.get(BorderModeOfTransportPage).exists(_.code != "5")
```

Comment:
Nice 👍 Can we add something like https://github.com/hmrc/ctc-departure-transport-details-frontend/blob/7917012c666a2a8edafea78dbdccbd2f45d7038d/app/config/Constants.scala#L45 to make this a bit clearer for someone picking this code up for the first time?